ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceNotifierTest.java

```
@@ -263,7 +269,9 @@ void onForkChoiceUpdated_shouldNotSendNotificationOfOrderedPayloadAttributes() {
 
     // current slot is 1
 
-    notifier.onForkChoiceUpdated(forkChoiceState); // calculate attributes for slot 2
+    notifier
+        .onForkChoiceUpdated(forkChoiceState)
+        .reportExceptions(); // calculate attributes for slot 2
```

We probably don't want to use `reportExceptions` in tests because it will log and ignore any exceptions.  Generally better to use `assertThat(...).isCompleted()` so that we know the operation did complete synchronously.